src/components/swapv2/LimitOrder/DeltaRate.tsx

```diff
@@ -8,45 +8,62 @@ import useTheme from 'hooks/useTheme'
 
 import { RateInfo } from './type'
 
-const DeltaRate = ({
+export function useGetDeltaRateLimitOrder({
```

Comment:
this hook have a lot of computations, please use memorize to prevent recalculate for each render